dlpython/main.go

```diff
@@ -43,6 +43,8 @@ var (
 
 // FindPythonConfig scans PATH for common python-config locations.
 func FindPythonConfig(customVersion string) (selectedVersion string, err error) {
+	logger.Debugf("Requested python version: %q", customVersion)
```

Comment:
I would use `WithField` instead of formatting, for additional fields, that way logs are parsable if needed.

Reply:
Logging improvements not in this scope :)